### PR TITLE
OSDOCS#12081: Adding OADP to OKE table

### PR DESCRIPTION
**Pull request by skopacz1**
[OSDOCS-12081](https://issues.redhat.com/browse/OSDOCS-12081) 

Version(s): 4.14+

This PR adds OADP to the list of features supported on OKE

QE review:
- [x] QE has approved this change.

Preview: [Feature summary](https://82403--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/oke_about.html#feature-summary)

### CODE REVIEW

File: welcome/oke_about.adoc [hunk 1]
```diff
@@ -286,6 +286,7 @@ s| Feature s| {oke} s| {product-title} s| Operator name
 | Embedded OperatorHub | Included | Included | N/A
 | Embedded Marketplace | Included | Included | N/A
 | Quay Compatibility (not included) | Included | Included | N/A
+| OpenShift API for Data Protection (OADP) | Included | Included | OADP Operator
```
Comment (skopacz1) [inline welcome/oke_about.adoc:289]: So far I only added it to the comprehensive list of features, but if OADP should also be added to [this seemingly higher-level list](https://docs.openshift.com/container-platform/4.16/welcome/oke_about.html#about_oke_similarities_and_differences) of similarities and differences, let me know.

Comment (weshayutin) [reply welcome/oke_about.adoc:289]: I like it the way it is, w/o the higher level list.

**Comment by weshayutin** [comment]
@akarol please provide a review and QE ack :)

**Comment by akarol** [comment]
LGTM

**Review by akarol — APPROVED** [review]

**Pull request merged**